### PR TITLE
this js file is needed

### PR DESCRIPTION
**Pull request by alksjfow32412**
Closes # 

## 📝 Description

> Add a brief description

## ⛳️ Current behavior (updates)

> Please describe the current behavior that you are modifying

## 🚀 New behavior

> Please describe the behavior or changes this PR adds

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+console.log('hi')
+
+// this is comment
```
Comment (FOSSgeek) [inline index.js:3]: remove this line and replace it with `console.log('comment')`

**Review by FOSSgeek — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (alksjfow32412) [reply index.js]: i did what you just said to me

Comment (alksjfow32412) [reply index.js]: now merge the pr plz

**Review by FOSSgeek — APPROVED** [review]
approved !!

**Pull request merged**